src/libraries/sendtohelix-browser.targets

```diff
@@ -250,7 +250,6 @@
         <Timeout Condition="'%(FileName)' == 'System.Collections.Immutable.Tests'">01:20:00</Timeout>
         <Timeout Condition="'%(FileName)' == 'System.Net.WebSockets.Client.Tests'">01:20:00</Timeout>
         <Timeout Condition="'%(FileName)' == 'System.Net.Http.Functional.Tests'">01:20:00</Timeout>
-        <Timeout Condition="'%(FileName)' == 'System.Memory.Tests'">01:20:00</Timeout>
```

Comment:
Isn't it possible that other timeouts can be reduced as well?

Reply:
`System.Collections.Immutable.Tests` were already running with `0` before.
The networking tests are just slow.

Reply:
Fine. `System.Private.Xml.Tests` does not have `0` and is not networking.

Reply:
okay, okay ... `System.Private.Xml.Tests` is down to [4 minutes](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-104152-merge-25618ece95014a09b7/WasmTestOnFirefox-ST-System.Private.Xml.Tests/1/console.03fa2160.log?helixlogtype=result)

Reply:
and `System.Text.Json.Tests` down to [10 minutes](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-104152-merge-25618ece95014a09b7/WasmTestOnFirefox-ST-System.Text.Json.Tests/1/console.8f05073c.log?helixlogtype=result)

Reply:
and `System.Collections.Immutable.Tests` down to [2 minutes](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-104152-merge-25618ece95014a09b7/WasmTestOnFirefox-ST-System.Collections.Immutable.Tests/1/console.187aaf40.log?helixlogtype=result)

Reply:
all of that on FF. I guess chrome would be ok as it also has jiterp.
I'm worried about V8 and NodeJS, but let's try.

Reply:
NodeJS is really slow for HTTP [Log](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-104361-merge-2247ccddd27a4bca99/WasmTestOnNodeJS-ST-System.Net.Http.Functional.Tests/1/console.3e46358c.log?helixlogtype=result)